### ISSUE TITLE
Scale must be greater than or equal to 1.

### ISSUE THREAD

**Issue by ahmetkapikiran** [PHPOffice/PhpSpreadsheet#227]
This is:

- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)

### What is the current behavior?
Scale must be greater than or equal to 1.

### What are the steps to reproduce?
```php
<?php

require __DIR__ . '/vendor/autoload.php';

$spreadsheet = IOFactory::load($this->csvPath);
$this->data = $spreadsheet->getActiveSheet()->toArray(null,true,true,true);
```

### Which versions of PhpSpreadsheet and PHP are affected?
development(0477e6fcfe)

### Temporary fix
https://github.com/ahmetkapikiran/PhpSpreadsheet/commit/b8efea41f00563366b75d91840b9656913ce1723?diff=unified

**Comment by PowerKiKi** [comment]
Code is incomplete and mentions CSV, but diff seems to be about XLSX. Description does not explain why the change is necessary. What happened before ?

**Comment by ksn135** [comment]
https://stackoverflow.com/a/46655754/1826164

**Comment by am0nshi** [comment]
@PowerKiKi hi, seems like problem is still actual in 2k19. Does it was solved in any kind ? I could attach problem xml to reproduce flow. Error directly is:
![image](https://user-images.githubusercontent.com/7633061/58945863-5a803180-878d-11e9-866e-6a1249ff640d.png)
Xlsx:
[Прайс_Радиоуправляемые машинки.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/3256470/_.xlsx)

**Comment by PowerKiKi** [comment]
I believe this is a duplicate of #350 and has been fixed in 2e37578971fea47644921d61e1887a7cc30df5c1 since v1.2.0
